### ISSUE TITLE
Local Module descriptor class for com.google.firebase.auth not found.

### ISSUE THREAD

**Issue by sanbnl** [firebase/quickstart-android#110]
### Step 1: Are you in the right place?

  * For issues or feature requests related to the code **in this repository** file a Github issue.
  * For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/) tagged appropriately.
  * For general Firebase discussion, use the [firebase-talk google group](https://groups.google.com/forum/#!forum/firebase-talk)
  * For help troubleshooting your application that does not fall under one of the above categories, reach out to the [personalized Firebase support channel](https://firebase.google.com/support/contact/troubleshooting/)

### Step 2: Describe your environment

  * Android device: _____
  * Android OS version: _____
  * Google Play Services version: _____
  * Firebase/Play Services SDK version: _____
  
#### Steps to reproduce:

  1. _____
  2. _____
  3. _____
  
#### Observed Results:

  * What happened?  This could be a description, `logcat` output, etc.
  
#### Expected Results:

  * What did you expect to happen?
  
#### Relevant Code:

  ```
  // TODO(you): code here to reproduce the problem
  ```

**Comment by sanbnl** [comment]
- Android device: Google Nexus 4 (Virtual)

- Android OS : 5.0

- Google Play Services version: 9.4.52

- Firebase/Play Services SDK version: 9.4.0


I downloaded the Zip file to try the quickstart in my environment, the app in installed in virtual device.
when i try to create a new in EmailPasswordActivity i get the following Error: **Local module descriptor class for com.google.firebase.auth not found.**
knowing that i have the app configured in the firebase console, and EmailPasswordAuthentification enabled.
I don't know why ????

**Comment by samtstern** [comment]
@sanbnl can you paste more of the logcat output from the device?

**Comment by sanbnl** [comment]
@samtstern  sure this is what happens when i set an email and a password and i click "CREATE ACCOUNT": 

D/EmailPassword: createAccount:sanbnl0016outlook.com
W/DynamiteModule: Local module descriptor class for com.google.firebase.auth not found.
W/DynamiteModule: Local module descriptor class for com.google.firebase.auth not found.
W/EGL_emulation: eglSurfaceAttrib not implemented
W/OpenGLRenderer: Failed to set EGL_SWAP_BEHAVIOR on surface 0xda8cde80, error=EGL_SUCCESS
D/EmailPassword: createUserWithEmail:onComplete:false
W/EGL_emulation: eglSurfaceAttrib not implemented
W/OpenGLRenderer: Failed to set EGL_SWAP_BEHAVIOR on surface 0xda8d0fa0, error=EGL_SUCCESS

**Comment by samtstern** [comment]
@sanbnl can you provide more logs?  I don't believe any of those messages is relevant to the problem.  Also can you attach an `onFailureListener` to your `createUserWithEmailAndPassword` call and log the resulting `Exception`?

**Comment by sanbnl** [comment]
@samtstern  Now i tried with a real device and it is working !!!!!
Thank you very much sam.

**Comment by samtstern** [comment]
@sanbnl glad you got it working!

**Comment by manism63** [comment]
I faced the same issue, I googled everywhere but couldn't find the exact answer, even if I was doing everything as documented by Google Android.  Suddenly after a day, my app was working fine and at that same moment I got the email from google heading "Google APIs Explorer connected to your Google Account". And it looks to me that google takes time to grant us access, just wait for this email, maybe this issue is not related to bad code.

**Comment by manism63** [comment]
I faced the same issue, I googled everywhere but couldn't find the exact answer, even if I was doing everything as documented by Google Android.  Suddenly after a day, my app was working fine and at that same moment I got the email from google heading "Google APIs Explorer connected to your Google Account". And it looks to me that google takes time to grant us access, just wait for this email, maybe this issue is not related to bad code.
